network/network_linux.go

```diff
@@ -98,6 +98,10 @@ func (nm *networkManager) newNetworkImpl(nwInfo *NetworkInfo, extIf *externalInt
 			return nil, fmt.Errorf("Ipv4 forwarding failed: %w", err)
 		}
 		logger.Info("Ipv4 forwarding enabled")
+		if err := networkutils.BlockEgressTrafficFromContainer(networkutils.AzureDNS, iptables.HTTPPort); err != nil {
+			return nil, fmt.Errorf("unable to insert vm iptables rule drop all wireserver port 80 packets: %w", err)
```

Comment:
Port 80 in the error message is from `iptables.HTTPPort` ? if so can we use the variable instead of hard coding.

Reply:
nit: Should we be using `errors.wrapf` (not sure but want to keep the cni files consistent going forward)?
@tamilmani1989

Reply:
> Port 80 in the error message is from `iptables.HTTPPort` ? if so can we use the variable instead of hard coding.

The insert ip table function calls the `ExecuteCommand` function (which logs the command itself) so I'll probably remove the details (like port or ip) from the error message altogether.

Reply:
yes we should errors.Wrapf .. @QxBytes  can you make that change?